deps/win/oibus_setup.iss

```diff
@@ -271,9 +272,9 @@ begin
     if FileExists(JsonFile) then
     begin
       ConfExists := True;
-      if MsgBox('An oibus.json file was found at ' + MyDataDir + '. Do you want to overwrite it?', mbInformation, MB_YESNO) = IDYES then
+      if MsgBox('An oibus.json file was found at ' + MyDataDir + '. Do you want to use it for your OIBus session ?', mbInformation, MB_YESNO) = IDNO then
       begin
-        if MsgBox('WARNING : Overwriting the current setup will delete all logins, passwords and data you saved so far.' + #13#10 + 'Are you sure you want to proceed?', mbInformation, MB_YESNO) = IDNO then
+        if MsgBox('WARNING : You are about to overwrite the current setup and delete all logins, passwords and data you saved so far.' + #13#10 + 'Are you sure you want to proceed ?', mbInformation, MB_YESNO) = IDNO then
```

Comment:
The aim is to have the default behaviour not to overwrite. In this case, keeping the origine question is ok if IDNO means that the default selected answer is NO.
Here you double negate the question which ultimately give the same behaviour than before.